CPlusPlus/index.html

```diff
@@ -1,5 +1,5 @@
 <html>
 <script>
-    location.href = 'https://docs.datalogics.com/CPlusPlus/APDFL18.0.4PlusP2a'
+    location.href = 'APDFL18.0.4PlusP2a'
```

Comment:
This one is resulting in a 404 error, as there's no P2a dir; should it be pointing at P1z?

Reply:
Shouldn't it point here: `https://docs.datalogics.com/CPlusPlus/` ?